File: hackerrank/easy/0044-mini-max-sum/python/solution.py
## solution.py

from typing import List

def miniMaxSum(arr: List[int]) -> None:
    assert len(arr) == 5, "Array must contain exactly 5 elements"
    s = sorted(arr)
    min_sum = sum(s[:4])
    max_sum = sum(s[1:])
    print(f"{min_sum} {max_sum}")

# --- demo run ---
if __name__ == "__main__":
    miniMaxSum([1, 2, 3, 4, 5])  # Expected: 10 14

File: hackerrank/easy/0044-mini-max-sum/swift/solution.swift
// solution.swift

import Foundation

// Sums four of five integers and prints minSum maxSum (space-separated).
// Uses Int64 for safe accumulation.
func miniMaxSum(arr: [Int]) {
    precondition(arr.count == 5, "Array must contain exactly 5 elements")
    let sorted = arr.sorted()
    let minSum: Int64 = sorted.prefix(4).reduce(0) { Int64($0) + Int64($1) }
    let maxSum: Int64 = sorted.suffix(4).reduce(0) { Int64($0) + Int64($1) }
    print("\(minSum) \(maxSum)")
}

// --- demo run ---
miniMaxSum(arr: [1, 2, 3, 4, 5])  // Expected: 10 14